common/onepanel/base/kustomization.yaml

```diff
@@ -7,9 +7,15 @@ resources:
  - add-label.yaml
  - add-network-policy.yaml
  - add-role.yaml
+ - add-service-account.yaml
+ - cluster-role.yaml
+ - cluster-role-binding.yaml
  - secrets.yaml
  - database-secret.yaml
+ - deploy-core-api.yaml
```

Comment:
we need to standardize naming on these. I see a mix of:

```
<action>-<k8s-resource-name>.yaml
<action>-<our-resource-name>.yaml
<k8s-resource-name>.yaml
```

Reply:
Agreed. Any preference?

Reply:
what about `<k8s-kind>-<k8s-metadata-name>.yaml`?  Example: `deployment-core.yaml`, `deployment-core-ui.yaml`. 

I think otherwise we may have duplicates. Also the action is determined by k8s when we run `apply` and could be `create/update`.

Reply:
I agree with the action not being included.
For the k8s-metadata-name, does that mean we'd split up core.yaml? Since it has a service?
So we'd have service-core.yaml, and deployment-core.yaml?

Reply:
Good question, ideally yes, even though we're not doing that in some other places.

Reply:
Okay, I'll split those out and see how it looks.

Reply:
@rushtehrani What about this case?
```apiVersion: v1
kind: Namespace
metadata:
  name: $(defaultNamespace) # replace with relevant namespace
  labels:
    istio-injection: enabled
    onepanel.io/enabled: "true"```

Reply:
@aleksandrmelnikov one idea is to just call it `namespace.yaml`, if there is another namespace in the folder, I assume it'll have a name and that will be `namespace-<meta.name>.yaml`.

Reply:
I don't think that'll work. What if we add another namespace with a placeholder for a name?
```name: $(auditNamespace)``` for example.

Maybe we do it like so?
```namespace-audit-namespace.yaml```?

Reply:
yup, that'll work better.

Reply:
How about: ```<k8s-kind>-<k8s-metadata-name>-<k8s-metadata-namespace>.yaml```?
Because I ran into overlap with configmaps.

Reply:
What about this one?
```apiVersion: builtin
kind: LabelTransformer
metadata:
  name: metadataLabelTransformer
labels:
  app.kubernetes.io/name: onepanel
  app.kubernetes.io/instance: onepanel-v0.5.0
  app.kubernetes.io/managed-by: onepanel-cli
  app.kubernetes.io/component: onepanel
  app.kubernetes.io/part-of: onepanel
  app.kubernetes.io/version: v0.5.0
fieldSpecs:
- path: metadata/labels
  create: true```

Reply:
makes sense, thinking if it makes sense to switch namespace and name: `<k8s-kind>-<k8s-metadata-namespace>-<k8s-metadata-name>.yaml`, since name is in namespace.

Reply:
I think we can leave those as they are, we'll only have one `LabelTransformer` per folder.

Reply:
Copy on LabelTransformer.

What do you mean by "name is in namespace"?
That looks like python code... :)

Reply:
I meant a resource by name of x is in namespace y

Reply:
I pushed up a refactoring, but it's kind-name-namespace.
How does it look?